index.js

```diff
@@ -26,7 +26,13 @@ Bonjour.prototype.find = function (opts, onup) {
 
 Bonjour.prototype.findOne = function (opts, cb) {
   var browser = new Browser(this._server.mdns, opts)
+  var tout = opts.timeout && setTimeout(function () {
+    browser.emit('notfound')
+    browser.stop()
+    if (cb) cb(false)
```

Comment:
I think it would make more sense to just call the callback without any arguments instead of `false` in case the timeout happens